{"name": "test_login_toast_wrong[case0]", "status": "passed", "attachments": [{"name": "log", "source": "8ea58560-f1c9-4337-a859-542938eab38d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'title': '手机号错误', 'phone': '17610772806', 'pwd': 'python', 'expected': '帐号或密码错误!'}"}], "start": 1603092654511, "stop": 1603092656180, "uuid": "fb89c81e-69b8-42f0-bae8-dda3d480c6c0", "historyId": "f6a128d3c7c56729a2ead33efa281e79", "testCaseId": "4ee7f628f4f90de8ceec763ad176e319", "fullName": "testcase.test_login.TestLogin#test_login_toast_wrong", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "ZBMAC-C02WD0XQH.local"}, {"name": "thread", "value": "15607-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}